packages/core/package.json

```
@@ -0,0 +1,73 @@
+{
+  "name": "@viamrobotics/prime-core",
+  "version": "0.0.1",
+  "scripts": {
+    "dev": "vite dev",
+    "build": "vite build && npm run package",
+    "preview": "vite preview",
+    "package": "svelte-kit sync && svelte-package && publint",
+    "prepublishOnly": "npm run package",
+    "check": "concurrently -g pnpm:check-*",
+    "check-svelte": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
+    "check-lint": "prettier --check . && eslint .",
+    "format": "prettier --write .",
+    "test": "vitest run",
+    "test:watch": "vitest"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "svelte": "./dist/index.js"
+    },
+    "./prime.css": "./src/prime.css",
+    "./theme.js": "./src/theme.js"
+  },
+  "files": [
+    "dist",
+    "!dist/**/*.test.*",
+    "!dist/**/*.spec.*",
+    "src/prime.css",
+    "src/theme.js"
+  ],
+  "peerDependencies": {
+    "svelte": "^4.0.0"
```

This is definitely a correct spec, but I like to spec peers with explicit bottom and top values, to make it easier in the future to simply widen the range, rather than mindlessly update the peer and accidentally introduce a breaking change to the package contract.

```suggestion
    "svelte": ">=4.0.0 <5"
```